scripts/linux/consts.bash

```diff
@@ -53,21 +53,29 @@ AGENT_JSON="$LOGZIO_TEMP_DIR/agent.json"
 
 ## Urls
 # Url for downloading jq binary
-JQ_URL_DOWNLOAD='https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64'
+JQ_URL_DOWNLOAD='https://github.com/stedolan/jq/releases/download/jq-1.7.1/jq-linux64'
```

Comment:
Can we rename it to JQ_AMD_URL_DOWNLOAD?
Same for the other links, and in mac as well :)